ckanext/xroad_integration/cli.py

```diff
@@ -26,7 +32,10 @@ def xroad():
 def init_db():
     """Creates the necessary tables in the database.
     """
-    utils.init_db()
+    import ckan.model as model
+    from ckanext.xroad_integration.model import init_table
```

Comment:
> Imports are always put at the top of the file, just after any module comments and docstrings, and before module globals and constants.

https://peps.python.org/pep-0008/#imports

However
> However, know when to be inconsistent – sometimes style guide recommendations just aren’t applicable. When in doubt, use your best judgment. Look at other examples and decide what looks best. And don’t hesitate to ask!

https://peps.python.org/pep-0008/#a-foolish-consistency-is-the-hobgoblin-of-little-minds

Is this a case where importing these here improves readability of the module enough to warrant making an exception to the style guide? Would it be enough to import the two globally as (for example) `ckan_model` and `xroad_integration_init_table`?